### DISCUSSION TITLE
Reload Device from CCU3 incl. Entitynames

### DISCUSSION THREAD

**Discussion by KlausH1967** [SukramJ/aiohomematic#1665] [Q&A]
Hello,
on the Read.me of "custom_homematic" it is described that you can use the service "homematicip_local.delete_device" to reload device and entity names from the CCU3.

However, I cannot find this service in the current version 1.64.0 of "Homematic(IP) Local".

![grafik](https://github.com/user-attachments/assets/13ccde1f-664c-4200-82ed-c008141a813d)

Is there another way to adopt new entity names of a device from the CCU3?
Thanks Klaus

**Comment by SukramJ** [comment]
Good to know that the [release notes](https://github.com/danielperna84/custom_homematic/releases/tag/1.64.0) are correct.
- Remove service delete_device. Use delete on the device entry instead

**Reply by KlausH1967** [reply]
Hello,
I'm glad I could help you here with a positive test ;-)

Thanks Klaus
